### PR TITLE
docs(blog): add cloudfront blog entry

### PR DESCRIPTION
**Pull request by CaptainCarpensir**
Adds blog entry for CloudFront for the v1.21 release

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/blogs/release-v121.en.md [hunk 1]
```diff
@@ -27,6 +27,38 @@ Copilot v1.21 brings several new features and improvements:
 
 ## CloudFront Integration
 
+One of our first major additions to the Copilot environment manifest! CloudFront is an AWS Content Delivery Network which helps people deploy their applications across the globe, and now you can enable a distribution by simply setting `cdn: true` in your environment manifest and running `copilot env deploy`.
+
+### Currently supported features:
+- A distribution deployed in front of your Public Load Balancer
+- Public ingress restricted to the CloudFront distribution
+- HTTPS traffic through an imported certificate, or a Copilot managed certificate
```
Comment (huanjani) [inline site/content/blogs/release-v121.en.md]:
```suggestion
- HTTPS traffic through an imported certificate, or a Copilot-managed certificate
```

**Review by huanjani — APPROVED** [review]
Just a teeny tiny nit; otherwise, looks great!

File: site/content/blogs/release-v121.en.md [hunk 2]
```diff
@@ -27,6 +27,38 @@ Copilot v1.21 brings several new features and improvements:
 
 ## CloudFront Integration
 
+One of our first major additions to the Copilot environment manifest! CloudFront is an AWS Content Delivery Network which helps people deploy their applications across the globe, and now you can enable a distribution by simply setting `cdn: true` in your environment manifest and running `copilot env deploy`.
```
Comment (iamhopaul123) [inline site/content/blogs/release-v121.en.md]:
```suggestion
One of our first major additions to the Copilot environment manifest! CloudFront is an AWS Content Delivery Network (CDN) which helps people deploy their applications across the globe, and now you can enable a distribution by simply setting `cdn: true` in your environment manifest and running `copilot env deploy`.
```

File: site/content/blogs/release-v121.en.md [hunk 3]
```diff
@@ -27,6 +27,38 @@ Copilot v1.21 brings several new features and improvements:
...
+
+### Currently supported features:
+- A distribution deployed in front of your Public Load Balancer
+- Public ingress restricted to the CloudFront distribution
```
Comment (iamhopaul123) [inline site/content/blogs/release-v121.en.md]:
```suggestion
- A distribution deployed in front of your public Application Load Balancer (ALB)
- ALB ingress restricted to the CloudFront distribution to protect from DDoS attack
```

File: site/content/blogs/release-v121.en.md [hunk 4]
```diff
@@ -27,6 +27,38 @@ Copilot v1.21 brings several new features and improvements:
...
+
+!!!info
+    CloudFront requires certificates to be in the `us-east-1` region. When importing a certificate, make sure to create your certificate in this region.
+
+First, create a certificate in the `us-east-1` region for your application with [AWS Certificate Manager](https://aws.amazon.com/certificate-manager/). You must add each domain associated with your application to this certificate. Once you've validated the certificate, you can add a field to your environment manifest to import the certificate for CloudFront:
+```yaml
+cdn:
+  certificate: arn:aws:acm:us-east-1:${AWS_ACCOUNT_ID}:certificate/13245665-h74x-4ore-jdnz-avs87dl11jd
+```
+Run `copilot env deploy`, then you can create an A-record in [Route 53](https://aws.amazon.com/route53/) which points to the CloudFront distribution created by Copilot. Just select to point the record to an `Alias` in the console, then Route 53 will show a valid DNS with any resources created in your account after you select the resource type (CloudFront in this case).
```
Comment (iamhopaul123) [inline site/content/blogs/release-v121.en.md]: If their environments are not in the same account as where their domains are hosted then this doesn't work. I think to be same we should suggest them to grab the CF distribution domain name and then use that as a value to create an A-record in the corresponding hosted zone.

**Review by iamhopaul123 — APPROVED** [review]

**Pull request merged**